src/raw/unix.rs

```diff
@@ -414,6 +418,32 @@ impl Socket {
     }
 
 
+    ///Sets whether this socket will be inherited to child porcesses or not.

```

Comment:
Please add note on a separate line that it uses `fcntl` to unset/set `FD_CLOEXEC`